File: app/src/main/java/com/fitpay/issuerdemo/MainActivity.java
package com.fitpay.issuerdemo;

import android.app.ActionBar;
import android.os.Bundle;
import android.widget.Toolbar;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;

public class MainActivity extends FragmentActivity {

    private BottomNavigationView bottomNavigationView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Toolbar toolbar = findViewById(R.id.tool_bar);
        setActionBar(toolbar);

        initViews();

        final AppToAppFragment appToAppFragment = new AppToAppFragment();
        final PushProvisioningFragment pushProvFragmentVisa = new PushProvisioningFragment();
        final PushProvisioningWebFragment pushProvFragmentMc = new PushProvisioningWebFragment();

        setCurrentFragment(appToAppFragment);

        bottomNavigationView.setOnNavigationItemSelectedListener(item -> {
            switch (item.getItemId()) {
                case R.id.app_to_app: setCurrentFragment(appToAppFragment); break;
                case R.id.push_prov: setCurrentFragment(pushProvFragmentVisa); break;
                case R.id.push_prov_custom: setCurrentFragment(pushProvFragmentMc); break;
            }
            return true;
        });
    }

    private void initViews() {
        bottomNavigationView = findViewById(R.id.bottom_navigation_view);
    }

    private void setCurrentFragment(Fragment fragment) {
        getSupportFragmentManager()
                        .beginTransaction()
                        .replace(R.id.main_fragment, fragment)
                        .commit();
    }

    public void setupActionBarTitle(String title) {
        ActionBar actionBar = getActionBar();
        if (actionBar != null) {
            actionBar.setTitle(title);
        }
    }
}

File: app/src/main/java/com/fitpay/issuerdemo/MainActivity.kt
package com.fitpay.issuerdemo

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private var appToAppButtonView: Button? = null
    private var pushProvButtonView: Button? = null
    private var ciqAppButtonView: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initViews()
    }

    private fun initViews() {
        appToAppButtonView = findViewById(R.id.app_to_app_button)
        pushProvButtonView = findViewById(R.id.push_prov_button)
        ciqAppButtonView = findViewById(R.id.ciq_app_button)

        appToAppButtonView?.setOnClickListener {
            val intent = Intent(this, AppToAppActivity::class.java)
            startActivity(intent)
        }

        pushProvButtonView?.setOnClickListener {
            val intent = Intent(this, PushProvisioningActivity::class.java)
            startActivity(intent)
        }

        ciqAppButtonView?.setOnClickListener {
            val intent = Intent(this, CIQSimulatorActivity::class.java)
            startActivity(intent)
        }
    }
}